components/eam/src/physics/cam/cldera_sai_tracers.F90

```diff
@@ -431,6 +431,14 @@ subroutine cldera_sai_tracers_timestep_tend(state, ptend, dt, ncol)
     use cam_abortutils,  only: endrun 
     use ref_pres,        only: ptop_ref, psurf_ref
     use mpishorthand
+#if defined(CLDERA_PROFILING)
+    use ppgrid,         only: begchunk
+use iso_c_binding, only: c_loc
+   use cldera_interface_mod, only: cldera_add_partitioned_field, max_str_len, &
```

Comment:
indentation is a little messed up around here. it's not a huge deal, but it makes it harder to read